parsimonious/tests/benchmarks.py

```diff
@@ -88,6 +88,6 @@ def test_not_really_json_parsing():
     seconds_each = total_seconds / NUMBER
 
     kb = len(json) / 1024.0
-    print 'Took %.3fs to parse %.1fKB: %.0fKB/s.' % (seconds_each,
+    print('Took %.3fs to parse %.1fKB: %.0fKB/s.' % (seconds_each,
```

Comment:
We might as well `from __future__ import print_function` at the top of the file so we get the full Python 3 semantics.